lib/wanda/execution/evaluation.ex

```diff
@@ -82,10 +92,14 @@ defmodule Wanda.Execution.Evaluation do
         message: "Fact gathering ocurred during the execution"
       }
     else
+      evalued_values = Enum.map(values, &eval_value(&1, facts, env))
```

Comment:
did you mean here `evaluated_values`?

Reply:
fixed here ad731d5d300b6400664d930fd4fb83bfcdb5ec63